src/app/src/util/util.js

```diff
@@ -453,3 +456,29 @@ export const makeFacilityListDataURLs = (id, count) => {
     return range(1, maxCount + 1)
         .map(page => makeFacilityListItemsRetrieveCSVItemsURL(id, page));
 };
+
+export const makeFacilityListSummaryStatus = (statuses) => {
+    const errorMessage = facilityListItemErrorStatuses.some(s => statuses.includes(s)) ?
+        facilityListSummaryStatusMessages.ERROR : '';
+    const awaitingMessage = [
+        facilityListItemStatusChoicesEnum.POTENTIAL_MATCH,
+    ].some(s => statuses.includes(s)) ?
+        facilityListSummaryStatusMessages.AWAITING : '';
+    const processingMessage = statuses.some(s => [
+        facilityListItemStatusChoicesEnum.UPLOADED,
+        facilityListItemStatusChoicesEnum.PARSED,
+        facilityListItemStatusChoicesEnum.GEOCODED,
+        facilityListItemStatusChoicesEnum.GEOCODED_NO_RESULTS,
+    ].includes(s)) ?
+        facilityListSummaryStatusMessages.PROCESSING : '';
+    const completeMessage = statuses.every(s => [
+        facilityListItemStatusChoicesEnum.MATCHED,
+        facilityListItemStatusChoicesEnum.CONFIRMED_MATCH,
+    ].includes(s)) ?
+        facilityListSummaryStatusMessages.COMPLETED : '';
```

Comment:
Not sure how to suggest cleaning it up, but this series of chained ternaries is a little hard to follow. I think the test does enough to verify that it's working, but in theory we could revise it with something like https://lodash.com/docs/4.17.11#cond that would do a form of pattern matching, which is I think what we want here.

Reply:
`cond` does a single match, but here we want multiple matches.

Reply:
Now that I'm looking at it again, this could also be a reduce operation that just rolls through the list and figures out what to do on each pass.

This is probably fine to leave as is assuming we're confident that the test is working, which it seems to be!

Reply:
👍